lib/hyrax/engine.rb

```diff
@@ -25,6 +25,11 @@ class Engine < ::Rails::Engine
       "Blacklight::Exceptions::RecordNotFound" => :not_found
     )
 
+    config.before_initialize do
+      # ActionCable should use Hyrax's connection class instead of app's
+      config.action_cable.connection_class = -> { 'Hyrax::ApplicationCable::Connection'.safe_constantize }
```

Comment:
Without this, the ActionCable server will *always* use `ApplicationCable::Connection` which is in the domain of the application, not Hyrax.